skops/hub_utils/_hf_hub.py

```diff
@@ -112,7 +114,7 @@ def init(*, model: Union[str, Path], requirements: List[str], dst: Union[str, Pa
     None
     """
     dst = Path(dst)
-    if dst.exists() and next(dst.iterdir(), None):
+    if dst.exists() and bool(next(dst.iterdir(), None)):
```

Comment:
aha? why do we need explicit bool here? This is proper python:

``` python
>>> if 3 or None:
...     print(1)
```

Reply:
The error is:

> skops/hub_utils/_hf_hub.py:117: error: Argument 1 to "next" has incompatible type "Generator[Path, None, None]"; expected "SupportsNext[bool]"
> skops/hub_utils/_hf_hub.py:117: error: Argument 2 to "next" has incompatible type "None"; expected "bool"

Reply:
That's very silly.